<launch>

    <node pkg="rosbot_controller" type="control_generator.py" name="control_generator" output="screen">
        <param name="control_mode"      value="periodic" />
        <param name="Tmax"              value="10" />
        <param name="period_lin"        value="3" />
        <param name="period_ang"        value="3" />
        <param name="v_min"             value="0.0" />
        <param name="v_max"             value="2.0" />
        <param name="w_min"             value="0.0" />
        <param name="w_max"             value="0.0" />
        <param name="a_lin"             value="1.0" />
        <param name="a_ang"             value="0.0" />
    </node>

    <include file="$(find logger)/launch/logger.launch">
        <arg name="output_folder"    value="my_dir" />
    </include>    

</launch>
